### PR TITLE
First draft

### PR DESCRIPTION
**Pull request by MikeBishop**
Made a first pass at some text. Thoughts?

### CODE REVIEW

**Review by richsalz — APPROVED** [review]
should put back the securty considerations section.

wow, really nice job.

**Comment by MikeBishop** [comment]
What don't you like about the Security Considerations?

**Comment by richsalz** [comment]
> What don't you like about the Security Considerations?

Nothing, my mind is going and I missed the "# Security Considerations" title I guess.  All good.  Merge it and/or wait for others to comment

File: draft-rsalz-httpapi-privacy.md [hunk 1]
```diff
@@ -46,21 +47,161 @@ informative:
...
+
+## Connection Blocking
+
+If an API request succeeds despite having an unencrypted endpoint configured,
+the developer or user is less likely to notice the misconfiguration. Where
+possible, it is advantageous for such a misconfiguration to fail immediately so
+that the error can be noticed and corrected.
+
+Servers MAY induce such an early failure by not accepting unencrypted
+connections, e.g. on port 80. This makes it impossible a client to send a
```
Comment (MikeBishop) [inline draft-rsalz-httpapi-privacy.md]:
```suggestion
connections, e.g. on port 80. This makes it impossible for a client to send a
```

**Comment by Acconut** [comment]
Thank you very much for writing this, Mike! The draft looks great and includes all points we have discussed so far. For me, this seems ready to be brought to the WG.

**Pull request merged**